addons/account/models/account_move.py

```diff
@@ -732,6 +732,22 @@ def _search_default_journal(self):
 
         return journal
 
+    def _search_made_sequence_hole(self, operator, value):
```

Comment:
Pur this function below `_compute_made_sequence_hole` (you are still in the middle of all compute functions anyways)

Refactor to use reuse the same query

```python
def _get_query_made_hole(self, ids=None):
    return f"""
            SELECT this.id
              FROM account_move this
              JOIN res_company company ON company.id = this.company_id
         LEFT JOIN account_move other ON this.journal_id = other.journal_id
                                     AND this.sequence_prefix = other.sequence_prefix
                                     AND this.sequence_number = other.sequence_number + 1
             WHERE other.id IS NULL
               AND this.sequence_number != 1
               AND this.name != '/'
              {"AND this.id = ANY(%(move_ids)s)" if ids is not None else ""}
    """, {'move_ids': ids}

def _compute_made_sequence_hole(self):
    query, params = self._get_query_made_hole(self.ids)
    self.env.cr.execute(query, params)
    made_sequence_hole = set(r[0] for r in self.env.cr.fetchall())
    for move in self:
        move.made_sequence_hole = move.id in made_sequence_hole

def _search_made_sequence_hole(self, operator, value):
    query, params = self._get_query_made_hole()
    operator = 'inselect' if operator == '=' else 'notinselect'
    return [('id', operator, (query, params))]
```

Reply:
Ok will do that